test/fake-hwclock.bats

```diff
@@ -0,0 +1,34 @@
+load test_helper
+export OS=$(uname -s)
+
+setup() {
+  if [ ! -f fake-hwclock.img ]; then
+    # download SD card image with cloud-init
+    curl -L -o download.img.zip https://github.com/hypriot/image-builder-rpi/releases/download/1.12.0-rc2/hypriotos-rpi-1.12.0-rc2.img.zip
+    unzip download.img.zip
+    # cut only 70 MByte to flash faster
+    dd if=hypriotos-rpi-1.12.0-rc2.img of=fake-hwclock.img bs=1048576 count=70
+  fi
+  stub_diskutil
+}
+
+teardown() {
+  umount_sd_boot /tmp/boot
+  rm -f $img
+  unstub_diskutil
+}
+
+@test "fake-hwclock: flash updates fake-hwclock.data" {
+  expected=$(TZ=UTC date '+%Y-%m-%d %H:%M')
```

Comment:
Remove seconds from expected string here, this hopefully runs the tests stable enough in CI without a change in the minute.